extension/test/integration/make-payment.handler.spec.js

```diff
@@ -1,97 +1,112 @@
 const { expect } = require('chai')
 
 const ctpClientBuilder = require('../../src/ctp')
-const iTSetUp = require('./integration-test-set-up')
-const constants = require('../../src/config/constants')
 const config = require('../../src/config/config')
+const constants = require('../../src/config/constants')
+const iTSetUp = require('./integration-test-set-up')
+
+describe('::make-payment with multiple store use case::', () => {
```

Comment:
Are you talking about testing with this endpoint? https://docs.commercetools.com/api/projects/stores

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/539/commits/7f2a208927784544332c79a28a52376eedc09f00